installer/templates/new/config/config.exs

```diff
@@ -6,9 +6,9 @@
 use Mix.Config
 
 # General application configuration
-config :<%= application_name %>,<%= if namespaced? do %>
-  app_namespace: <%= application_module %>,<% end %>
-  ecto_repos: [<%= application_module %>.Repo]
+config :<%= application_name %><%= if namespaced? do %>,
+  app_namespace: <%= application_module %><% end %><%= if ecto do %>,
+  ecto_repos: [<%= application_module %>.Repo]<% end %>
 
```

Comment:
If the app isn't namespaced and not using ecto, we'll be left with `config :my_app` where we should have generated nothing